src/Settings/Mods.ts

```diff
@@ -309,9 +309,13 @@ export function GenerateModsSettings(): GameStartr.IModAttachrCustoms {
                         for (const actor of opponent.actors) {
                             actor.level += playerPokemonAvg - enemyPokemonAvg;
 
-                            for (const statistic of statistics) {
-                                (actor as any)[statistic] = (actor as any)[statistic + "Normal"] =
-                                    this.MathDecider.compute("pokemonStatistic", actor, statistic);
+                            if (this.utilities.shouldEvolve(actor)) {
+		                        this.utilities.evolvePokemon(actor);
```

Comment:
There's a tab on that line. 

![image](https://cloud.githubusercontent.com/assets/3335181/19629838/4e586ed2-9932-11e6-95c5-f8ebe337fe66.png)